.gitmodules

```diff
@@ -17,3 +17,6 @@
 	path = third_party/zlib
 	url = https://github.com/madler/zlib
 	ignore = dirty
+[submodule "plugins/mission/libs/json11"]
+	path = plugins/mission/libs/json11
```

Comment:
Let's have it in `plugins/mission/third_party/json11` instead, to be consistent with the rest of the project.